src/main/java/carpet/script/CarpetExpression.java

```diff
@@ -2061,23 +2061,28 @@ else if ("!spectating".equalsIgnoreCase(playerName))
         });
 
         this.expr.addLazyFunction("entity_area", 7, (c, t, lv) -> {
```

Comment:
Then the number arguments can suddenly be variable. Seems you haven't try that since that would fail if you supply with less than 7 args, right?

Reply:
oopsi silly me